p2p/enode/node.go

```diff
@@ -184,6 +189,11 @@ func (n *Node) TCP() int {
 	return int(n.tcp)
 }
 
+// QUIC returns the QUIC port of the node.
+func (n *Node) QUIC() int {
```

Comment:
I think it should return `uint16`.

Reply:
I copied the `UDP()` and `TCP()` return types. But I can update them to have `uint16` as return type.

https://github.com/ethereum/go-ethereum/blob/83e70aa3d00bbee9713b31768a30a7741eee8945/p2p/enode/node.go#L177-L185